src/Assortment/PackagingType.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Kollex\Assortment;
+
+use InvalidArgumentException;
+use UnhandledMatchError;
+
+enum PackagingType
+{
+    case CA;
+    case BX;
+    case BO;
+
+    public static function resolve(string $type): PackagingType
+    {
+        $type = mb_strtolower($type);
+        if (str_contains($type, ' ')) {
+            $type = substr(strtolower($type), 0, strpos($type, ' '));
+        }
+
+        try {
+            return match ($type) {
+                'bottle' => PackagingType::BO,
+                'box' => PackagingType::BX,
+                'case' => PackagingType::CA
+            };
+        } catch (UnhandledMatchError $e) {
```

Comment:
It makes sense to catch exceptions and log incorrect data to the log file, but we don't need to stop our processing if some of our rows are incorrect.

In our case, we are getting a fatal error every time when package type is incorrect for some reason or doesn't exist in our match condition.